bridget/sap_flow/sap_flow.py

```diff
@@ -0,0 +1,107 @@
+"""
+Sap Flow package
+----------------
+
+These functions can be used to process sap flow measurements to 
+evapotranspiration estimations. Each of the function will return the output 
+along with estimated/calculated uncertainties. To suppress the output, 
+set the with_uncertainties parameter to False.
+
+"""
+
+def heat_pulse_velocity(temperatures, m_c, rho_b, probe_spacing, with_uncertainties=True):
+    """Calculate heat pulse velocity
+
+    [Description]
+
+    Parameters
+    ----------
+    temperatures : list, numpy.array
+        Array of coupled temperatures
+    m_c : list, numpy.array
+        Water content of the wood
+    rho_b : list, numpy.array
+        Sapwood density
+    probe_spacing : int
+        Spacing of the probes, that measured the temperatures
+    with_uncertainties : bool
+        If True (default), all estimated uncertainties will be returned 
+        along with the data
+
+    Returns
+    -------
+    vh : numpy.array
+        Array of heat pulse velocity. 
+    t_rise : numpy.array
+        timing offset in maximum temperature rise. 
+        Will only be returned if ``with_uncertainties=True``.
+    var_mc : numpy.array
+        variance in m_v
+        Will only be returned if ``with_uncertainties=True``.
+    var_rho : numpy.array
+        variance in rho_b
+        Will only be returned if ``with_uncertainties=True``.
```

Comment:
Are these actually time-series? In order to calculate a variance over time it either needs to be applied in a moving window, or the whole function is reducing the input data.
I would need some clarification here.